src/styles/index.css

```diff
@@ -0,0 +1,111 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  width: 198px;
+  height: 406px;
```

Comment:
The width and height of the `.card` class should be defined in terms of rem units to ensure better scalability and readability of the code. For example, if the base font size is 16px, then `width: 198px` can be converted to `width: 12.375rem`.